src/sql/gazetteer_export_func.sql

```diff
@@ -14,7 +14,7 @@
 set search_path=gazetteer_export, gazetteer, public;
```

Comment:
Instead of specifying it in the create function statement could you just change this line to `SET search_path=gazetteer, public;` ? This makes it consistent with the pattern in the other sql files.

Reply:
Remove `gazetteer_export`? this was left as the function heavily modifies this schema. Though it does appear reference to objects in the `gazetteer_export` do reference the schema. I will test this

Reply:
updated. 
* gazetteer_export search path removed. 
* gazetteer schema reference removed